src/alerting/components/AlertingIndex.tsx

```diff
@@ -78,7 +79,10 @@ const AlertingIndex: FunctionComponent = () => {
                 <Icon glyph={IconFont.BookPencil} />
                 Now you can use Notebooks to explore your data while building an
                 alert
-                <Link to="/notebook/from/notification" onClick={recordClick}>
+                <Link
+                  to={`/${PROJECT_NAME.toLowerCase()}/from/notification`}
```

Comment:
thanks for all these

Reply:
don't thank me. I didn't do it for tasks